BungeeCord-Patches/0048-Speed-up-some-common-exceptions.patch

```diff
@@ -172,19 +172,19 @@ index 237955ab..d0bd4d75 100644
 +    // Waterfall end
  }
 diff --git a/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java b/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
-index 460a79a1..b0caf6d6 100644
+index ccc80506..d8d65093 100644
 --- a/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
 +++ b/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
-@@ -15,6 +15,8 @@ import java.util.Set;
- import java.util.UUID;
- import java.util.logging.Level;
- import javax.crypto.SecretKey;
+@@ -21,6 +21,8 @@ import javax.crypto.SecretKey;
+ import lombok.AllArgsConstructor;
+ import lombok.Data;
+ import lombok.EqualsAndHashCode;
 +import javax.crypto.spec.SecretKeySpec;
```

Comment:
This import change seems to be misplaced. Previously it was fitting in alphabetical order.